cpu/cc3200/ldscripts/cc3200.ld

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (C) 2019 Wlad Meixner
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/* Memory Space Definitions: */
+MEMORY
+{
+    rom (rx)    : ORIGIN = 0x20004000, LENGTH = 32K
+    ram (w!rx)  : ORIGIN = 0x2000C000, LENGTH = 240K - 32K
```

Comment:
Maybe add a comment here that the MCU can't run off internal flash, but instead copies the contents of the flash to RAM on startup and that it's possible to adjust the RAM/Flash split if needed.